db/schema_v12.go

```diff
@@ -0,0 +1,76 @@
+package db
+
+import "fmt"
+
+type v12Schema struct{}
+
+func (s v12Schema) Deploy(db *DB) error {
+	var err error
+
+	jobs, err := db.GetAllJobs(&JobFilter{})
+	if err != nil {
+		return nil
+	}
+	for _, job := range jobs {
+		healthy := job.LastTaskStatus == "done"
+		db.UpdateJobHealth(job.UUID, healthy)
+	}
+	tenant := RandomID()
```

Comment:
Why are we setting a random UUID as a default for targets?

Reply:
I was running the tests for targets and the tests fail because the UUID was not assigned. I was also looking at the schema_v4 and saw that tennant_uuid is set to a random UUID as a default.